### PR TITLE
Miracles of george (4)

### PR DESCRIPTION
**Pull request by SusanneHummel**
Here are four more new miracles contained in BDLaethb4. Unfortunately, another six are in a separate PR

For the former issue on creating those files: BetaMasaheft/Documentation#1491

Concering the titles:
The first title is our own, the second and third titles are taken from the catalogues by Strelcyn and Grebaut/Tisserant.

### CODE REVIEW

File: new/LIT6218MGMoney.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6218MGMoney" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: St George punishes a man who
+                    denied to pay back the money from the priest</title>
```
Comment (Gnisci) [inline new/LIT6218MGMoney.xml:9]: refused to pay back to a priest

File: new/LIT6218MGMoney.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                    <ab>፲ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወበመዋዕሊሁ፡ ለቈስጠንጢኖስ፡ ጻድቅ፡ ሀሎ፡ ውስተ፡
+                        ቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ማር፡ ጊዮርጊስ፡ ፩ ኅፁፀ፡ ምሕረት፡ ዘኢይሁብ፡ ለአግብርተ፡ ቤተ፡ ክርስቲያን፡ ዘልማዶሙ፡
+                        ተሴስዩ፡ እምንዋየ፡ ቤተ፡ ክርስቲያን፡</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወመጽኡ፡ ኀበ፡ ቤተ፡ ክርስቲያኑ፡ ነገሥት፡
+                            ወዐበይት፡ እምኵለሄ። ወአምጽኡ፡ ንዋየ፡ ብዙኃ፡ ወመሳቀለ፡ ዘወርቅ፡ ወብሩር፡ ወአጽሕልተ፡
+                            ወጽዋዓተ፡ ወመጻሕፍተ፡ ለአንብቦቶን። ጸሎቱ፡ ወበረከቱ፡ ወኃይለ፡ ረድኤቱ፡ የሀሉ፡
+                            ምስለ፡<gap reason="ellipsis" resp="SH"/>
```
Comment (MassimoVilla) [inline new/LIT6218MGMoney.xml:98]: I would leave a space between  ምስለ፡ and `<gap reason="ellipsis" resp="SH"/>`

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you!!

**Comment by SusanneHummel** [comment]
Thank you! I updated the file.

**Review by MassimoVilla — APPROVED** [review]

File: new/LIT6218MGMoney.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፲ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወበመዋዕሊሁ፡ ለቈስጠንጢኖስ፡ ጻድቅ፡ ሀሎ፡ ውስተ፡
+                        ቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ማር፡ ጊዮርጊስ፡ ፩ ኅፁፀ፡ ምሕረት፡ ዘኢይሁብ፡ ለአግብርተ፡ ቤተ፡ ክርስቲያን፡ ዘልማዶሙ፡
+                        ተሴስዩ፡ እምንዋየ፡ ቤተ፡ ክርስቲያን፡</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወመጽኡ፡ ኀበ፡ ቤተ፡ ክርስቲያኑ፡ ነገሥት፡
+                            ወዐበይት፡ እምኵለሄ። ወአምጽኡ፡ ንዋየ፡ ብዙኃ፡ ወመሳቀለ፡ ዘወርቅ፡ ወብሩር፡ ወአጽሕልተ፡
```
Comment (thea-m) [inline new/LIT6218MGMoney.xml:96]: መሳቅለ?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: new/LIT6218MGMoney.xml (hunk 3 above)
Comment (SusanneHummel) [reply new/LIT6218MGMoney.xml]: መሳቀለ
I changed it

**Review by thea-m — APPROVED** [review]

**Pull request merged**